_infra/helm/frontstage/templates/deployment.yaml

```diff
@@ -235,5 +235,7 @@ spec:
             value: "{{ .Values.gnu.recipient }}"
           - name: PASSWORD_RESET_ATTEMPTS_TIMEOUT
             value: "{{ .Values.password.attemptTimeout }}"
+          - name: PROFILER_ENABLED
+            value: "{{- if .Values.frontstageProfiler.enabled -}}True{{- else -}}False{{- end -}}"
```

Comment:
[frontstageProfiler:
  enabled: true](https://github.com/ONSdigital/ras-rm-config/pull/236#pullrequestreview-1739976749)

Reply:
Makes sense. This is updated.